examples/step-32/step-32.cc

```diff
@@ -3577,9 +3577,9 @@ namespace Step32
                                 EquationData::R1,
                                 (dim==3) ? 96 : 12,
                                 true);
-    static HyperShellBoundary<dim> boundary;
-    triangulation.set_boundary (0, boundary);
-    triangulation.set_boundary (1, boundary);
+    triangulation.set_all_manifold_ids(0);
+    static SphericalManifold<dim> manifold;
+    triangulation.set_manifold (0, manifold);
```

Comment:
Did you test this? In ASPECT, we got into trouble because we use a `MappingQ(4)` which requires a boundary object and can't deal with manifold objects :-(